packages/broker/src/plugins/operator/closeExpiredFlags.ts

```diff
@@ -0,0 +1,33 @@
+import { EthereumAddress, Logger } from '@streamr/utils'
+import { ContractFacade, SponsorshipResult } from './ContractFacade'
+
+const logger = new Logger(module)
+
+export const closeExpiredFlags = async (
+    flagLifetime: number,
+    operatorContractAddress: EthereumAddress,
+    contractFacade: ContractFacade
+): Promise<void> => {
+    logger.info('running closeExpiredFlags')
+
+    const minFlagStartTime = Math.floor(Date.now() / 1000) - flagLifetime
+
+    const sponsorships = (await contractFacade.getSponsorshipsOfOperator(operatorContractAddress))
+        .map((sponsorship: SponsorshipResult) => sponsorship.sponsorshipAddress)
```

Comment:
This mapping is needed only at line 22, could move it there

Reply:
I'll leave it, since it won't save anything, because in the length=0 case the mapping won't run anyway.